sonar-project.properties

```diff
@@ -1,8 +1,8 @@
 ### service-bot sonarqube plugin managed file
 sonar.coverage.exclusions=**/test/**/*,**/tests/**/*,**/mock/**/*,**/mocks/**/*,**/*mock*,**/*test*
-sonar.coverage.jacoco.xmlReportPaths=**/jacoco.xml
+sonar.coverage.jacoco.xmlReportPaths=**/jacoco-report/jacoco.xml,**/jacoco-report-it/jacoco.xml
 sonar.cpd.exclusions=**/test/**/*,**/tests/**/*,**/mock/**/*,**/mocks/**/*,**/*mock*,**/*test*
-sonar.exclusions=**/*.pb.*,**/mk-files/**/*,**/test/**/*
+sonar.exclusions=**/*.pb.*,**/mk-files/**/*,**/test/**/*,**/static/templates/**/*
 sonar.java.binaries=.
```

Comment:
These changes would be made by ServiceBot given the `service.yml` changes, but I wanted it to take effect in this PR.